web-components/navbar/navbar-btn.js

```diff
@@ -0,0 +1,44 @@
+const navbarBtn = document.createElement('template')
+navbarBtn.innerHTML = `
+<style>
+  @import "web-components/navbar/navbar-btn.css";
+</style>
+<button aria-label="Menu Button">
+  <div></div>
+  <div></div>
+  <div></div>
+  <div></div>
+  <div></div>
+</button>
+`
+
+class NavbarBtn extends HTMLElement {
+    constructor (breakpoint) {
+        super()
+        this.attachShadow({ mode: 'open' })
+        this.shadowRoot.appendChild(navbarBtn.content.cloneNode(true))
+        this.breakpoint = breakpoint
+    }
+
+    connectedCallback () {
+        const btn = this.shadowRoot.querySelector('button')
+
+        btn.addEventListener('click', () => {
+            btn.classList.toggle('change')
```

Comment:
`change` in not specific enough as a class name. The name should clearly say what it is for.